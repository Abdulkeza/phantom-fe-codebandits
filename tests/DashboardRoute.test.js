import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import DashboardRoute from '../src/DashboardRoutes.js';

describe('Dashboard route testing', () => {
  it('Should render the dashboard routes', () => {
    const wrapper = renderer
      .create(
        <BrowserRouter>
          <DashboardRoute />
        </BrowserRouter>
      )
      .toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
