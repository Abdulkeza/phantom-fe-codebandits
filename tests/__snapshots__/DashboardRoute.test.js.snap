// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard route testing Should render the dashboard routes 1`] = `null`;
